includes/definitions/ciena-rls.yaml

```diff
@@ -0,0 +1,28 @@
+os: ciena-rls
+text: 'Ciena Reconfigurable Line System'
+icon: ciena
+over:
+    - { graph: device_bits, text: Traffic }
+type: network
+mib_dir: ciena
+discovery_modules:
+    bgp-peers: false
```

Comment:
Is there a reason to disable these ? The rule is to keep everything enabled unless it breaks something.